cli/batch/interfaces.py

```diff
@@ -0,0 +1,21 @@
+"""
+Provide implementation of the batch interfaces.
+"""
+
+
+class BatchInterface:
+    """
+    Implements batch interface.
+    """
+
+    async def get_list(self, query=None):
+        """
+        Get a list of batches from REMChain.
+
+        Arguments:
+            query (dict, optional): dictionary with specific parameters
+
+        Returns:
+            List of batches.
```

Comment:
Apply to each method you have — https://github.com/Remmeauth/remme-core-cli/pull/18#discussion_r277605811

Reply:
Done.